src/idvc/dvc_interface.py

```diff
@@ -455,12 +455,23 @@ def CreateHelpPanel(self):
             "8. To restart the mask creation, click on 'Clear mask'.\n"
             "9. Each mask can be selected and reloaded by clicking on 'Load Saved Mask'.")
         
-        self.help_text.append("Dense point clouds that accurately reflect sample geometry and reflect measurement objectives yield the best results.\n"
+        self.help_text.append("A point cloud for the DVC analysis can be generated or imported from file.\n"
+            "Dense point clouds that accurately reflect sample geometry and reflect measurement objectives yield the best results.\n"
             "The first point in the cloud is significant, as it is used as a global starting point and reference for the rigid translation between the two images.\n"
-            "If the point 0 you selected in image registration falls inside the mask, then the pointcloud will be created with the first point at the location of point 0.\n"
-            "If you load a pointcloud from a file, you must still specify the subvolume size on this panel, which will later be input to the DVC code.\n"
-            "It will be the first point in the file that is used as the reference point.")
-
+            "The reference point for the DVC analysis is the first point in the point-cloud file.\n\n"
+            "Generate a point cloud:\n"
+            "1. A 3D point cloud will be created across the entire extent of the mask unless the 2D dimensionality is selected i.e. the point cloud is created on the currently displayed slice of the image.\n" 
+            "2. The overlap is the percentage overlap of the subvolume regions. Values larger than 0.2 are not recommended.\n"
```

Comment:
0.2? is that true?

Reply:
It should be true but I deleted the sentence.